tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -78,8 +85,10 @@ spec:
           exit 1
         fi
 
+        set +x
         echo "${pyxisCert}" > /tmp/crt
         echo "${pyxisKey}" > /tmp/key
+        set -x
```

Comment:
I'm all good with adding this, but it isn't clear to me why you want to add the `set -x` (and then `set +x` for the if elif checks at the start)

Reply:
Yeah yeah, I get why we don't want `set -x` for the cert echoes. I was just saying your PR adds `set -x` for some very simple if statements then turns it off on line 89. It would be pretty much the same to just only add it after the cert echoes. Either way, it is a nit pick